src/consts/network.ts

```diff
@@ -7,6 +7,6 @@ export const WALLET_URL = PRODUCTION ? 'https://wallet.provenance.io' : 'https:/
 export const ROOT_NAME = 'wallettest3.pb'
 export const FEE_DENOM = 'nhash'
 
-// export const BASE_URL = 'http://localhost:13459/service-invoice/v1'
+// export const BASE_URL = 'http://localhost:13459/v1'
```

Comment:
Swapped this a bit because the backend now doesn't explicitly specify `/service-invoice` when it spins up locally